xla/service/gpu/gpu_hlo_schedule.cc

```diff
@@ -431,7 +431,7 @@ static int64_t GetSchedulerMemoryLimit(
 
 absl::StatusOr<ScheduleMetadata> ScheduleGpuModule(
     HloModule* module, int64_t pointer_size,
-    const se::DeviceDescription& gpu_device_info) {
+    const se::DeviceDescription& gpu_device_info, bool run_accuracy_checker) {
```

Comment:
Instead of a bool, could we make this a flag in xla.proto , have it the correct default, and consult module options?

Reply:
Done